ansible/roles/operator-pipeline/templates/openshift/tasks/digest-pinning.yml

```diff
@@ -79,3 +80,23 @@ spec:
           echo "Manifests are pinned."
           echo -n "false" | tee $(results.dirty_flag.path)
         fi
+    - name: verify-related-images
+      image: "$(params.pipeline_image)"
+      workingDir: $(workspaces.source.path)
+      script: |
+        #! /usr/bin/env bash
+        set -xe
+
+        BUNDLE_PATH=$(realpath $(params.bundle_path))
+        cat references.json
+        REFERENCE_COUNT=$(jq length references.json)
+        CSVFILE=$(find $BUNDLE_PATH -name "*clusterserviceversion.yaml" -o -name "*clusterserviceversion.yml")
+        RELATED_IMAGE_COUNT=$(yq -e '.spec.relatedImages | length' $CSVFILE)
+
+        if [[ $REPLACEMENT_COUNT == $RELATED_IMAGE_COUNT ]]; then
+          echo "Related images section exist."
```

Comment:
Minor grammar change: "exist" -> "exists"

Reply:
Thanks for catching that. I changed it.